### PR TITLE
Managed vaults

### PR DESCRIPTION
**Pull request by Patricie29**
This PR introduces: 
- a first demo of the new page for Official and Community vaults 
- table with demo data for both official and community vaults 
- info and warning modal when creating/depositing into community vault 

UPDATED DEMO:

https://github.com/user-attachments/assets/a26d1541-a73c-464b-bcc9-7d96474ff91f

### CODE REVIEW

File: src/components/common/Intro.tsx [hunk 1]
```diff
@@ -52,6 +52,18 @@ function IntroBackground(props: { bg: Props['bg'] }) {
...
+        <div className='absolute top-0 right-0 block w-140 opacity-5'>
+          <GridTire />
+        </div>
+      )
+    case 'vaults-community':
+      return (
+        <div className='absolute top-0 right-0 block w-140 opacity-5'>
+          <GridLandscape />
+        </div>
+      )
```
Comment (linkielink) [inline src/components/common/Intro.tsx:66]: Group the cases
```
    case 'borrow':
      return (
        <div className='absolute top-0 right-0 block w-140 opacity-5'>
          <GridHole />
        </div>
      )
    case 'lend':
    case 'vaults-official':
      return (
        <div className='absolute top-0 right-0 block w-180 opacity-5'>
          <GridTire />
        </div>
      )
    case 'farm':
    case 'vaults-community':
      return (
        <div className='absolute top-0 right-0 block w-140 opacity-5'>
          <GridLandscape />
        </div>
      )
    case 'hls-farm':
      return (
        <div className='absolute bottom-0 right-0 block w-140 opacity-10'>
          <GridWeb />
        </div>
      )
    case 'hls-staking':
      return (
        <div className='absolute top-0 right-0 block w-110 opacity-10'>
          <GridPlanet />
        </div>
      )
```

File: src/components/header/navigation/Routes.tsx [hunk 2]
```diff
@@ -35,6 +37,8 @@ export default function Routes() {
         <Route path='/v1' element={<V1Page />} />
         {chainConfig.hls && <Route path='/hls-staking' element={<HLSStakingPage />} />}
         {chainConfig.hls && <Route path='/hls-farm' element={<HLSFarmPage />} />}
+        <Route path='/vaults-official' element={<VaultsOfficialPage />} />
```
Comment (linkielink) [inline src/components/header/navigation/Routes.tsx:40]: `vaults-official` should be `vaults`

The `vaults-community` is just an overview page. 

The final URLS should be:
`vaults`
`vaults/create`
`vaults/{VAULT_ADDRESS}`
`vaults/{VAULT_ADDRESS}/mint-account`
`vaults/{VAULT_ADDRESS}/delete`
etc...

**Review by linkielink — CHANGES_REQUESTED** [review]
Please also add the yellow "You're entering dangerous teritory" dialog that shows up, whenever a user enters the `vaults-community` page.

**Comment by linkielink** [comment]
This PR was done in the managed-vaults branch.
For future work, please create branches based on the managed-vaults branch and then do PRs for the managed-vaults branch. This PR will be closed, as it won't be merged into main or develop until Managed Vaults are feature complete.

**Comment by Patricie29** [comment]
yep, will do. I actually thought this might not be the correct approach. @linkielink

**Pull request closed without merging**